paper/paper.md

```diff
@@ -1,63 +1,54 @@
 ---
-title: 'JointSurvivalModels.jl: Numeric approach to joint models'
+title: 'JointSurvivalModels.jl: A Julia package for general Bayesian joint models'
 tags:
-  - Julia
-  - Joint Model
-  - biostatistics
   - Survival analysis
-  - Bayesian statistics
+  - Nonlinear
+  - Biostatistics
+  - Time-to-event
+  - Mixed-effects model
 authors:
   - name: Yannik Ammann
+    orcid: 0009-0009-3296-3577
     affiliation: "1, 2"
-  - name: Adrian M. Price-Whelan
-    orcid: 0000-0000-0000-0000
-    equal-contrib: true
-    affiliation: "1, 2" # (Multiple affiliations must be quoted)
-  - name: Author Without ORCID
-    equal-contrib: true # (This is how you can denote equal contributions between multiple authors)
+  - name: Daniel Sabanés Bové
     affiliation: 2
-  - name: Author with no affiliation
-    corresponding: true # (This is how to denote the corresponding author)
+  - name: Francois Mercier
+    orcid: 0000-0002-5685-1408
     affiliation: 2
-  - given-names: Ludwig
-    dropping-particle: van
-    surname: Beethoven
+  - name: Douglas O. Kelkhoff 
+    orcid: 0009-0003-7845-4061 
     affiliation: 2
+  
 affiliations:
  - name: ETH Zürich, Switzerland
    index: 1
  - name: Hoffmann-La Roche Ltd.
    index: 2
-date: 02 February 2022
+date: 02 February 2024
 bibliography: paper.bib
-
-# Optional fields if submitting to a AAS journal too, see this blog post:
-# https://blog.joss.theoj.org/2018/12/a-new-collaboration-with-aas-publishing
-aas-doi: 10.3847/xxxxx <- update this with the DOI from AAS once you know it.
-aas-journal: Astrophysical Journal <- The name of the AAS journal.
 ---
 
 # Summary
-This software implements a numerical approach to define a distribution based on the description of the hazard function from survival analysis. In particular this allows to define joint models of time-to-even data and longitudinal measurements. Using numerical integration, the likelihood of events can be calculated, allowing Bayesian inference frameworks to sample the posterior distribution of the model's parameters. Additionally, this implementation is able to generate samples of joint models. This allows its use in simulations, which are common in Bayesian workflows [@BayesianWorkflow].
+`JointSurvivalModels.jl` implements a numerical approach to define a distribution based on the hazard function. In particular, this provides a mechanism for defining joint models of time-to-event data and longitudinal measurements. Using numerical integration, the likelihood of events can be calculated, allowing Bayesian inference frameworks to sample the posterior distribution of the model's parameters. Additionally, this implementation can generate samples of joint models. This allows its use in simulations and predictions, which are common in Bayesian workflows [@BayesianWorkflow].
```

Comment:
so when we say here "likelihood can be calculated", I think we should be able to use maximum likelihood inference too right? That would make the paper overall stronger if we can include that, as well as a small example (or at least an example in the package) maybe.

Reply:
Yes in general a numeric approch to optimization should still work the same with the caveat that complex joint modles might not have a concave likelihood so the optimisation could be local or not converge. I could write:

"This allows Bayesian inference frameworks to sample the posterior distribution of the model's parameters or numerical optimizers to calculate the MLE."

Reply:
yeah that could be nice. can you also still try to use such an optimization approach in the example to see if it could work in practice?

Reply:
I think https://turing.ml/v0.22/docs/using-turing/guide#maximum-likelihood-and-maximum-a-posterior-estimates would be very easy to try out because we can stay with the Turing framework and just it for maximum likelihood estimation

Reply:
Yes, I will take a look at this and if it works out incoporate it in the example and somehow in the text! Thanks

Reply:
that will be great, thanks!
and even if the maximum likelihood does not work, it could be interesting to show that you can optimize the posterior density (which is a "regularized" likelihood) and obtain the maximum posteriori estimates with optimization